lib/active_admin/views/components/table_for.rb

```diff
@@ -115,10 +115,8 @@ def render_data(data, item)
       end
 
       def is_boolean?(data, item)
-        if item.respond_to? :has_attribute?
-          item.has_attribute?(data) &&
-            item.column_for_attribute(data) &&
-            item.column_for_attribute(data).type == :boolean
+        if item.class.respond_to? :columns_hash
+          column = item.class.columns_hash[data.to_s] and column.type == :boolean
```

Comment:
@seanlinsley There was already a discussion about that in #3841. I think the 120 chars from #3841 are to much, but 90-100 could be ok. In the past I have seen 80 chars as a "soft limit", in cases where a code rewrite is ugly then a long line, I have preferred the long line.